docs/ECLLanguageReference/ECLR_mods/WkFlo-GLOBAL.xml

```diff
@@ -72,12 +72,14 @@
 
   <programlisting>I := RANDOM() : INDEPENDENT;  //calculated once, period
 G := RANDOM() : GLOBAL;       //calculated once in each graph
+A := RANDOM();                //calculated each time used
 
-ds := DATASET([{1,0,0},{2,0,0}],{UNSIGNED1 rec,UNSIGNED Ival, UNSIGNED Gval });
+ds := DATASET([{1,0,0,0},{2,0,0,0}],{UNSIGNED1 rec,UNSIGNED Ival, UNSIGNED Gval , UNSIGNED Aval });
 
 RECORDOF(ds) XF(ds L) := TRANSFORM
   SELF.Ival := I;
   SELF.Gval := G;
+  SELF.Aval := A;
```

Comment:
While strictly speaking the change is correct, I hesitate slightly to merge it because I'm not 100% sure it is the way people would expect it to work, and there is an outside possibility that it might change in the future.

Better to say

SELF.AVal := RANDOM();

then it would be clear to everyone that it should be a new value each time.